.github/workflows/unit-tests.yml

```diff
@@ -31,20 +31,21 @@ jobs:
         env:
```

Comment:
Changed the location of the `.env` file that is created, to flow with new monorepo folder structure.